layers/+tools/transmission/packages.el

```diff
@@ -29,50 +29,94 @@
     :defer t
     :init (progn
             (spacemacs/set-leader-keys "att" 'transmission)
+
             (spacemacs/declare-prefix-for-mode 'transmission-mode "ma" "add")
             (spacemacs/declare-prefix-for-mode 'transmission-mode "mg" "goto")
             (spacemacs/declare-prefix-for-mode 'transmission-mode "ms" "set")
             (spacemacs/declare-prefix-for-mode 'transmission-mode "mt" "toggle")
+            (spacemacs/declare-prefix-for-mode 'transmission-mode "mQ" "Queue")
             (spacemacs/set-leader-keys-for-major-mode 'transmission-mode
-              "gt" 'transmission-files
-              "at" 'transmission-add
-              "sd" 'transmission-set-download
-              "gp" 'transmission-peers
-              "gi" 'transmission-info
-              "sl" 'transmission-set-ratio
+              "D"  'transmission-delete
               "m"  'transmission-move
-              "r"  'transmission-remove
-              "ts" 'transmission-toggle
+              "q"  'transmission-quit
+              "R"  'transmission-remove
+              "v"  'transmission-verify
+              "S"  'tabulated-list-sort
+
+              ;; add
               "aa" 'transmission-trackers-add
+              "at" 'transmission-add
+
+              ;; goto
+              "gi" 'transmission-info
+              "gp" 'transmission-peers
+              "gt" 'transmission-files
+
+              ;; set
+              "sb" 'transmission-set-bandwidth-priority
+              "sd" 'transmission-set-download
+              "sl" 'transmission-label
+              "sr" 'transmission-set-ratio
               "su" 'transmission-set-upload
-              "v"  'transmission-verify
-              "q"  'transmission-quit
-              "sb" 'transmission-set-bandwidth-priority)
+
+              ;; toggle
+              "tm" 'transmission-toggle-mark
+              "ts" 'transmission-toggle
+              "tt" 'transmission-turtle-mode
+
+              ;; Queue
```

Comment:
How about
J/K move down/move up
H/L move to top/move to bottom

Reply:
I thought of doing this but I was unsure if it would okay for me to use the movement keys here. If we're using HJKL should it be nested under a prefix like "SPC m Q H" "SPC m Q J" ... or would "SPC m H" "SPC m J" ... be enough?

Reply:
well, putting them under `SPC m` is still very cumbersome.
How about define them as evil normal state keys?
`(evil-define-key 'normal transmission-mode-map ..)`

Reply:
Also, please move this entire chunk except `(spacemacs/set-leader-keys "att" 'transmission)` to config, so that they're executed only when `transmission` is loaded.

Reply:
I will make the changes soon.

Reply:
I've moved everything but the binding for "att" to `:config` and changed the queue manipulation keys to use HJKL. Please check if they're in order.